hack/e2e.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-if [ "$COZYSTACK_INSTALLER_YAML" = "" ]; then
+if [ '$COZYSTACK_INSTALLER_YAML' = '' ]; then
```

Comment:
Unfortunately this is not going to work, because we need to check if this variable is not empty.

Your suggestion will check not this content of the variable but text `$COZYSTACK_INSTALLER_YAML` which is always not null.

Please consider something like:
```suggestion
if [ -z "$COZYSTACK_INSTALLER_YAML" ]; then
```


Which shell do you use?

Reply:
i m using zsh on arch linux  6.9.9-arch1-1